pkg/pillar/hypervisor/qmp.go

```diff
@@ -15,12 +15,24 @@ import (
 const sockTimeout = 10 * time.Second
 
 func execRawCmd(socket, cmd string) ([]byte, error) {
-	monitor, err := qmp.NewSocketMonitor("unix", socket, sockTimeout)
+	var retry int = 3
+	logrus.Infof("executing QMP command: %s", cmd)
+	var err error
+	var monitor *qmp.SocketMonitor
+
+	for retry >= 0 {
```

Comment:
Can you please elaborate why is this necessary?

Reply:
I saw QMP connection failures, supposedly due to a late qemu start

Reply:
Move to a different PR